### PR TITLE
MGT login sample in React

### PR DESCRIPTION
**Pull request by GuidoZam**
|        Q        |                    A                    |
| --------------- | --------------------------------------- |
| Bug fix?        | no |
| New feature?    | no |
| New sample?     | yes |

## What's in this Pull Request?

Added a sample about using the Login control in a SPFx solution.

### CODE REVIEW

**Review by sebastienlevert — APPROVED** [review]
Thanks for the contribution! This looks great! Want to get a final approval from our code wizard @gavinbarron before we merge!

**Review by gavinbarron — APPROVED** [review]
Awesome work! 

LGTM!

**Pull request merged**
